### PR TITLE
docs(accordion): implement and document vars and attrs

### PR DESCRIPTION
**Pull request by lalithkarikelli**
## Description

Implement and Document variables and attributes

### What are the relevant story cards/tickets? Any additional PRs or other references?

Jira: SURF 2094

## Before you request a review for this PR:

- [x] For UI changes, did you manually test in recent versions of modern browsers (Chrome, Firefox, and Safari)?
- [x] For UI changes, did you manually test in IE11 and legacy Edge?
- [ ] Did you add component tests for any new code?
- [ ] Did you run the component unit tests via `yarn test` to ensure all tests passed?
- [ ] Did you include a screenshot of the component tests?
- [ ] If you changed/added functionality, did you update the demo page and documentation?
- [ ] If needed, did you add or modify the demo test page to test the changed/added functionality?
- [x] Did you assign reviewers?
- [x] In Jira, have you linked to this PR on the ticket(s)?

### CODE REVIEW

**Review by BY00565233 — APPROVED** [review]
DEV LGTM

File: src/scss/components/accordion/_stepper.scss [hunk 1]
```diff
@@ -25,44 +25,44 @@ hx-accordion.hxStepper {
       // TODO: clarify mouse hover/click behavior with Design
```
Comment (100stacks) [inline src/scss/components/accordion/_stepper.scss:25]: Please update this block:

https://github.com/HelixDesignSystem/helix-ui/blob/28f00a163c8b114bac182408c21635867b7a133e/src/scss/components/accordion/_stepper.scss#L9-L14

**Review by 100stacks — CHANGES_REQUESTED** [review]

File: src/scss/components/accordion/_stepper.scss (hunk 1 above)
Comment (lalithkarikelli) [reply src/scss/components/accordion/_stepper.scss:25]: Changes have been done, please review.

**Review by 100stacks — CHANGES_REQUESTED** [review]

File: src/scss/components/accordion/_stepper.scss [hunk 2]
```diff
@@ -7,10 +7,10 @@ hx-accordion.hxStepper {
   counter-reset: step;
 
   hx-accordion-panel {
-    border-top: 1px solid $gray-300;
+    border-top: 1px solid var(--hxAccordian-panel-borderColor, $gray-300);
```
[the remark this answers is not in the document]
Comment (lalithkarikelli) [reply src/scss/components/accordion/_stepper.scss]: Changes have been done, please review.

File: src/scss/components/accordion/_stepper.scss [hunk 3]
```diff
@@ -7,10 +7,10 @@ hx-accordion.hxStepper {
...
 
     &:last-child {
-      border-bottom: 1px solid $gray-300;
+      border-bottom: 1px solid var(--hxAccordian-child-borderColor, $gray-300);
```
[the remark this answers is not in the document]
Comment (lalithkarikelli) [reply src/scss/components/accordion/_stepper.scss]: Changes have been done, please review.

**Review by 100stacks — APPROVED** [review]
Dev LGTM

**Pull request merged**